README.md

```diff
@@ -51,12 +51,12 @@ import SnapshottingTests
 class DemoAppPreviewTest: SnapshotTest {
```

Comment:
In this README the existing sample uses `SnapshotTest` and that works in my own separate project to perform the Preview layout test. 

Should DemoAppPreviewTest.swift file be changed to inherit from `SnapshotTest` instead of `PreviewLayoutTest`?

Reply:
There are two tests in that example, one that uses SnapshotTest and one that uses PreviewLayoutTest

https://github.com/EmergeTools/SnapshotPreviews/blob/main/Examples/DemoApp/DemoAppTests/DemoAppSnapshotTest.swift
https://github.com/EmergeTools/SnapshotPreviews/blob/main/Examples/DemoApp/DemoAppTests/DemoAppPreviewTest.swift

You can use either, the snapshot test one just creates an image (a snapshot) and the layout one just performs a layout pass but doesn't save a png

Reply:
Brilliant and makes sense, thank you. From here I see "2 workflows awaiting approval. This workflow requires approval from a maintainer." and a blocking "Only those with write access to this repository can merge pull requests" so please merge whenever you like.